main.go

```diff
@@ -223,8 +224,12 @@ func main() {
 	mux.Handle("/", http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		found := len(cfg.allowPaths) == 0
 		for _, path := range cfg.allowPaths {
-			if req.URL.Path == path {
-				found = true
+			found, err = pathmodule.Match(path, req.URL.Path)
+			if err != nil {
+				klog.Fatalf("Allowed path pattern is not correct: %s", path)
```

Comment:
This will exit the whole process with error code 255 at runtime. Note this is a handler not bootstrapping code. Please simply `return` in this case.

According to https://pkg.go.dev/path#Match:
> The only possible returned error is ErrBadPattern, when pattern is malformed.

We could leverage this and iterate before the handler in the bootstrapping main goroutine:
```
for _, path := range cfg.allowPaths {
  _, err := pathmodule.Match(path, req.URL.Path)
  if err != nil {
    klog.Fatalf("Failed to verify allow path: %s", path)
  }
}
```

Reply:
thanks, I will replace the klog.Fatalf by a simple return.

Reply:
check for allowPaths and ignorePaths are added,too.